### PR TITLE
Added AEM 6.3 support for conditional hiding in edit dialogs

### PR DESCRIPTION
**Pull request by RolandDahlemPV**
hiding dialog fields based on condition does not work in AEM 6.3.
This Pull Request fixes the issue

### CODE REVIEW

File: content/src/main/content/jcr_root/apps/acs-commons/touchui-widgets/showhidedialogfields/source/showhidedialogfields.js [hunk 1]
```diff
@@ -82,6 +82,10 @@
     	if(select){
     		return "select";
     	}
+	    
+	if($(element).hasClass("coral-Checkbox")){
```
Comment (justinedelson) [inline content/src/main/content/jcr_root/apps/acs-commons/touchui-widgets/showhidedialogfields/source/showhidedialogfields.js]: (commented on wrong PR)
Per the comment at the top of https://helpx.adobe.com/experience-manager/6-3/sites/developing/using/reference-materials/coral-ui/coralui3/Coral.Checkbox.html, this class shouldn't be referenced directly. I believe the correct check would be on the element name.

Comment (RolandDahlemPV) [reply content/src/main/content/jcr_root/apps/acs-commons/touchui-widgets/showhidedialogfields/source/showhidedialogfields.js]: @justinedelson:

Comment (justinedelson) [reply content/src/main/content/jcr_root/apps/acs-commons/touchui-widgets/showhidedialogfields/source/showhidedialogfields.js]: According to https://stackoverflow.com/questions/8852665/how-can-i-get-name-of-element-with-jquery, you should use attr

Comment (RolandDahlemPV) [reply content/src/main/content/jcr_root/apps/acs-commons/touchui-widgets/showhidedialogfields/source/showhidedialogfields.js]: But "./frame" has nothing to do with checkbox. I don't think we can identify a checkbox with this in a reliable way

Comment (justinedelson) [reply content/src/main/content/jcr_root/apps/acs-commons/touchui-widgets/showhidedialogfields/source/showhidedialogfields.js]: Sorry, I edited my comment above to clarify that you should look at the element name, not the name property (or attribute). Strictly speaking, in DOM terms this is "tag name".

Comment (RolandDahlemPV) [reply content/src/main/content/jcr_root/apps/acs-commons/touchui-widgets/showhidedialogfields/source/showhidedialogfields.js]: I changed it to use tag name

**Comment by justinedelson** [comment]
Thanks!

**Pull request merged**